src/01-dinosaur-facts.js

```diff
@@ -22,7 +22,28 @@ const exampleDinosaurData = require("../data/dinosaurs");
  *  getLongestDinosaur(dinosaurs);
  *  //> { Brachiosaurus: 98.43 }
  */
-function getLongestDinosaur(dinosaurs) {}
+function getLongestDinosaur(dinosaurs) {
+  //empty arr
+  let longestDino;
+  let longDino = 0;
+  if (dinosaurs.length === 0) { return {}; }
```

Comment:
Another way to do this is to initiate your `longestDino` variable with an empty object. That way if the `dinosaurs` array is empty it will not execute the for loop and will return the object on line 44.